core/src/main/java/tech/pegasys/eth2signer/core/http/handlers/SignForIdentifierHandler.java

```diff
@@ -51,11 +51,11 @@ public SignForPublicKeyHandler(
   @Override
   public void handle(RoutingContext routingContext) {
     final RequestParameters params = routingContext.get("parsedParameters");
-    final String publicKey = params.pathParameter("publicKey").toString();
-    final Optional<ArtifactSigner> signer = signerProvider.getSigner(publicKey);
+    final String identifier = params.pathParameter("identifier").toString();
+    final Optional<ArtifactSigner> signer = signerProvider.getSigner(identifier);
     if (signer.isEmpty()) {
-      LOG.error("Unable to find an appropriate signer for request: {}", publicKey);
-      routingContext.fail(404);
+      LOG.trace("Unable to find an appropriate signer for request: {}", identifier);
```

Comment:
log message is probably more like "Unsuitable handler, invoking next"?

Reply:
It's not necessarily that it's inappropriate. The key might not exist at all.

Reply:
but if you do a context.next, then this handler can't make a global statement about the appropriateness of the identifier or not - only that _this_ handler can't handle it

Reply:
done